### DISCUSSION TITLE
Hi Team, idea about design

### DISCUSSION THREAD

**Discussion by amirabayoumi** [HYF-Class22/agile-development-group4-hyf-redesign#7] [Ideas]
@betulaydinturkk @Mamine777 since we can add more than one page - what do you think to make 3 pages - the most important sections for our user persona ! 
so if i add sponser for example : i will rebuild the section of funding us .....!! 
What do you think, every body take 1 user and add backlog add design add page , so everthing for this user !

**Comment by betulaydinturkk** [comment]
I will think about student and I add my decision here. If everybody finish before the meeting we can talk all things about.
